toolkit/tools/imager/imager.go

```diff
@@ -403,15 +403,24 @@ func fixupExtraFilesIntoChroot(installChroot *safechroot.Chroot, config *configu
 	return
 }
 
-func cleanupExtraFilesInChroot(installChroot *safechroot.Chroot, config configuration.SystemConfig) (err error) {
+func cleanupExtraFiles() {
 	dirsToRemove := []string{additionalFilesTempDirectory, postInstallScriptTempDirectory, sshPubKeysTempDirectory}
+
 	for _, dir := range dirsToRemove {
-		err = os.RemoveAll(dir)
+		logger.Log.Infof("Cleaning up directory %s", dir)
+		err := os.RemoveAll(dir)
 		if err != nil {
 			logger.Log.Errorf("Failed to cleanup directory (%s). Error: %s", dir, err)
-			return
 		}
 	}
+}
+
+func cleanupExtraFilesInChroot(chroot *safechroot.Chroot) (err error) {
+	logger.Log.Infof("Proceeding to cleanup extra files in chroot %s.", chroot.RootDir())
+	err = chroot.Run(func() error {
+		cleanupExtraFiles()
```

Comment:
Hides error.  If temporary files are left in the image this should constitute a build breka.

Reply:
I'll warn since it's in setup chroot.